guitar-api.yaml

```
@@ -0,0 +1,122 @@
+openapi: 3.0.3
+info:
+  title: Guitar API
+  description: |-
+   Reference Basic infromation on guitar models and the musicians that have used them at various points in their careers.
+    - [Guitar API Repository](https://github.com/Bobs-code/Guitar-API)
+  contact:
+    email: robertgriffintm@gmail.com
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+servers:
+  - url: http://localhost:8080
+tags:
+  - name: guitar
+    description: Everything about popular guitar models
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+paths:
+  /guitar:
+    get:
+      tags: 
+        - guitar
+      summary: find a guitar by ID
+      description: returns a single guitar
+      operationId: getGuitarByID
+      parameters:
+        - name: guitarId
```

```suggestion
        - name: guitar_id
```

Generally speaking, `camelCase` vs `snake_case` is pure preference. Both are valid and fine. But URLs are case-insensitive, so generally `path` params should always be `snake_case` that way you’re not dealing with `guitarid` (which is valid)

---

Cool, this is helpful. Struggled to balance level of consistency across API code and documentation. For some reason I thought "Since my Go API Method is `getGuitarById()`, therefore all other references to it must be camel case."